<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>catalogDiscount</class>
 <widget class="QDialog" name="catalogDiscount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>284</width>
    <height>134</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>284</width>
    <height>134</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>284</width>
    <height>134</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Параметры скидки для товара</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Артикул:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineArt">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineProduct">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Товар:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Спец. цена:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineSpecPrice">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Скидка (%):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineDiscount">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineWithDiscount">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Со скидкой:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushApply">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Применить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
